addons/mail/static/src/scss/composer.scss

```diff
@@ -439,6 +444,7 @@ $o-mail-attachment-margin: 5px;
 
             .o_attachment_delete_cross {
                 color: white;
+                float: none;
```

Comment:
I don't think this style makes sense.
The cross icon should be in the top-right corner for all attachments, including images.

Reply:
@alexkuhn Removed `float: none`.
Now cross icon will appear on the top-right corner for images, like all other attachments. (which is used to appear on the top-left corner)